doc/JupyterBook_Admin.md

```diff
@@ -0,0 +1,74 @@
+# Jupyter-Book Administration Notes
+
+The documents (markdown and executable notebooks) in this repo are meant to be the source material from
+which a _static_ website can be built using [jupyter book](https://jupyterbook.org/en/stable/intro.html).
+
+## Build Notes
+
+* JB will execute notebooks in order to render them and extract an HTML representation.
+* The execution environment must, therefore, contain all of the packages and compute resources needed
+  for that to succeed (cluster hardware, S3 write access, python modules/libraries, etc).
+* The environment must also contain the jupyter-book package. This package is not needed for any
+  of the notebooks to run; it is only needed for the build process.  For this reason, it is not included
+  in the `HyTEST.yml` environment file.
+* JB is based on [Sphinx](https://jupyterbook.org/en/stable/explain/sphinx.html) as the documentation engine.
+  Sphinx uses templates to establish styling and
+  features.  The Project Pythia Cookbook style is the basis of the HyTEST book style.
+* Styling templates are installed as python packages, usin conda or pip.  Much like the jupyter-book
+  package itself, `sphinx_pythia_theme` must be installed.  This package is also not included in the
+  `HyTEST.yml` environment file.
+* Jupyter Book only builds the static website; it does not publish to a server.
+  - How to publish depends on where you want the docs to go.
+  - The default for this repo is to use 'GitHub Pages' as the deployment.
+  - Publication to GH Pages is made easier with `ghp-import`.  This is yet another of those packages that
+    your build environment will need, but is not included in the `HyTEST.yml` file (because notebook users
+    don't need it).
+* To ease the jupyter-book command invocation, a `Makefile` is included, with simplified targets defined:
+  - `make book` -- builds the static website to local disk.  Use your browser to open a `file://path/` URL
```

Comment:
I just installed these packages and tried to run `make book` from the top level of the hytest repo. It told me there was "No rule to make target 'book'." Is this because the Makefile is missing?